### DISCUSSION TITLE
Dashboard: Shared dashboard data source and ability to pull data from multiple panels

### DISCUSSION THREAD

**Discussion by vellala2000** [grafana/grafana#38334] [Data Sources]
**What would you like to be added**:

Looks like currently we are restricted to only one panel query when using --dashboard datasource. Add ability to import data from more than one panel  query just like we are using any other data source like --mixed-- or influxDB

**Why is this needed**:

Sometime there is a requirement to consolidate stats from all individual panel of the dashboard and show a quick overall status for executive reporting. this would be helpful instead writing to avoid duplicate and complex queries to data source.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
